### PR TITLE
Travel page!

### PR DESCRIPTION
**Pull request by Hour-of-the-Owl**
yay?

### CODE REVIEW

File: docs/pages/travel.md [hunk 1]
```diff
@@ -4,4 +4,20 @@ layout: default
...
 # Fast Travel
 
-*work in progress*
+## Overview
+
+The Travel window allows you to travel directly to any outpost in the game, so long as you've already unlocked that outpost.
+
+## Usage
+
+Select an outpost from the drop-down menu and you will instantly travel there. You can type the first letter of the outpost to find it in the list faster. The word "The" is not considered for alphabetical order.
```
Comment (HasKha) [inline docs/pages/travel.md]: this is misleading. you can type as much as you want. Also add that you can navigate the list with arrows up/down.

File: docs/pages/travel.md [hunk 2]
```diff
@@ -4,4 +4,20 @@ layout: default
...
+
+There are also 8 default outpost buttons to choose from. If you want more, you can add your favourite outposts, using the dropdown menus at the bottom. You can change the number of favourites in [Settings](settings).
+
+If you want to go to a particular district, make sure you choose it from the district dropdown menu.
+
+There is also an option in [Settings](settings) to make the Travel window close when you travel, although this is off by default.
+
+## Bug
```
Comment (HasKha) [inline docs/pages/travel.md]: "known bug" or issue is better

**Comment by Hour-of-the-Owl** [comment]
get better at commenting please Has

**Comment by HasKha** [comment]
thanks

**Pull request merged**